plugins/metrics/parser/src/metricsparser.cpp

```diff
@@ -51,6 +51,40 @@ std::vector<std::string> MetricsParser::getDependentParsers() const
   return std::vector<std::string>{};
 }
 
+bool MetricsParser::preparse()
+{
+  if(!_fileIdCache.empty())
+  {
+    try
+    {
+      (util::OdbTransaction(_ctx.db))([this] {
```

Comment:
Vexparse -> uniform

Reply:
Fixed in 4301dea623f06401b259435c3999d35a3f646dd6.